Assets/Augmented Tyria/Scripts/Utils/MumbleCamera.cs

```diff
@@ -16,7 +16,9 @@ private void Awake()
 
     private void Update()
     {
-        this.Camera.fieldOfView = this.Mumble.Link.GetIdentity().Fov * Mathf.Rad2Deg;
+		var identity = this.Mumble.Link.GetIdentity();
+		if( identity != null )

```

Comment:
When is the identity ever null? When the game isn't running?

Reply:
Exactly.

Reply:
And does this single check prevent all exceptions when the game isn't running?

Reply:
As far as I could check, it did.